.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "resolve-chrome-uri"]
```

Comment:
suggestion: This might need documenting (if it's not already) to properly pull in submodules. I know I have hit this stumbling block on onboarding with other projects.

Reply:
Good point. I'll add a mention to the README.